apps/web/src/pages/templates/workflow/StepActiveSwitch.tsx

```diff
@@ -15,8 +31,9 @@ export const StepActiveSwitch = ({ control, index }) => {
           <StyledSwitch
             {...field}
             disabled={readonly}
-            checked={value}
-            label={`Step is ${value ? 'Active' : 'Disabled'}`}
+            checked={check}
+            onChange={handleChecked}
+            label={`Step is ${check ? 'Active' : 'Not Active'}`}
```

Comment:
Hey, I updated the switch label to 'Step is Not Active' as it is on Figma. do you think it is a good idea or should i return it to how it was before 'Step is Disabled'